app/src/main/java/com/example/springprojectstationbuild/MainActivity.kt

```diff
@@ -21,40 +19,42 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContent {
             viewModel.fetchComputerComponents()
-            //val components by viewModel.components.observeAsState(initial = emptyList())
```

Comment:
Good to remove this.
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.